settings.gradle

```diff
@@ -40,6 +40,8 @@ include ':smoke-tests:wildfly'
 include ':smoke-tests:springboot'
 
 // instrumentation:
+include ':instrumentation:akka-context-propagation'
+include ':instrumentation:akka-context-propagation:akka-testing'
```

Comment:
Is there any benefit to having this separate testing module?

Reply:
This is one strange module. I think I will merge it indeed...